streams/bo.m3u

```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
 https://solo.disfrutaenlared.com:1936/tuch/ngrp:tuch_all/playlist.m3u8
 #EXTINF:-1 tvg-id="XTOTV.bo" status="online",XTOTV (404p) [Not 24/7]
 http://www.channel.tevemi.com:1935/XtoTv/XtoTv/playlist.m3u8
+#EXTINF:-1 tvg-id="UPPTV.bo" status="online",UPP TV (480p) [Not 24/7]
+https://d2byzrnrdb259r.cloudfront.net/hls/upptv.m3u8
```

Comment:
doesn't work for me

Reply:
it answers but gives a black screen when not airing online.

Reply:
now it works.

Reply:
![snapshot](https://user-images.githubusercontent.com/30985701/188337190-6c44aaa0-4817-4393-b18f-748c0e9ae1a6.jpg)

Reply:
works thanks